--- "a/.\\pngjs\\util\\Table.js"
+++ "b/.\\canvaskit\\lib\\js\\new\\Table.js"
@@ -315,3 +315,23 @@ const Table = {
 };
 
 module.exports = Table;
+
+if (typeof Benchmark !== "undefined") {
+    Benchmark.getTable = (style = "doubleVertical", extraSpaces = 1, ...includeSum) => {
+        const data = Benchmark.getAll(...includeSum, false);
+
+        const keys = Object.keys(data),
+            times = Object.values(data);
+
+        const columns = {
+                name: "Name:",
+                time: "Time (ms):"
+            },
+            rows = {
+                name: keys,
+                time: times
+            };
+
+        return Table.drawTable(columns, rows, style, extraSpaces);
+    };
+}
