--- "a/.\\canvaskit\\libs\\js\\original\\vips.js"
+++ "b/.\\canvaskit\\libs\\js\\new\\vips.js"
@@ -24,7 +24,7 @@ var Vips = (() => {
             ja = [],
             ka = "./this.program",
             la;
-        ea ? (la = self.location.href) : da ? (la = _scriptName) : (la = void 0);
+        la = self.location.href;
         var ma = "";
         function na(a) {
             return l.locateFile ? l.locateFile(a, ma) : ma + a;
@@ -639,7 +639,7 @@ var Vips = (() => {
             Qe = [],
             Ge = {};
         function Re() {
-            for (var a = 6 < navigator.hardwareConcurrency ? navigator.hardwareConcurrency : 6; a--; ) Ne();
+            for (var a = 1 < navigator.hardwareConcurrency ? navigator.hardwareConcurrency : 1; a--; ) Ne();
             Za(() => {
                 fb("loading-workers");
                 Se(() => gb("loading-workers"));
@@ -5029,7 +5040,7 @@ var Vips = (() => {
         ci.push(0, 1, void 0, 1, null, 1, !0, 1, !1, 1);
         l.count_emval_handles = () => ci.length / 2 - 5 - bi.length;
         Za(() => {
-            Yg.VIPS_MAX_THREADS = 6 < navigator.hardwareConcurrency ? navigator.hardwareConcurrency : 6;
+            Yg.VIPS_MAX_THREADS = 1 < navigator.hardwareConcurrency ? navigator.hardwareConcurrency : 1;
             Yg.VIPS_CONCURRENCY = 1;
         });
         Qa.unshift(() => {
